cypress/integration/play_word_and_card_spec.js

```diff
@@ -1,6 +1,7 @@
 describe('Play word and card', () => {
     beforeEach(() => {
         cy.login('username');
+        cy.wait(500);
```

Comment:
Do you want to move this into the startGame function itself so we don't have to repeat the wait everywhere?